src/components/BaseTimer.js

```diff
@@ -52,12 +52,13 @@ class BaseTimer extends Component {
     this.timer = setInterval(() => {
       let newTime;
       if (countDown) {
-        newTime = this.state.currentTime - 10;
+        newTime = this.state.currentTime - 10; // eslint-disable-line react/destructuring-assignment
```

Comment:
Suppressing this for now. As far as I can tell, this is a style rule and not necessarily a functional one. I need to reference the variable in state because I am updating state in this function. Since I am destructuring state in the outer function, a bug is introduced if I use the destructured version.